app/src/renderer/components/wallet/LiCoin.vue

```diff
@@ -0,0 +1,88 @@
+<template lang="pug">
+.li-coin
```

Comment:
@fedekunze what is your thinking re: `coin` vs. `token` or `denom`?

Reply:
My order of preferences:
`LiCoin` > `LiCoinBalance` > `LiToken`

Reply:
can you explain your thinking a little bit? to my knowledge, this is the first time we use `coin` anywhere - usually we use `denom` - `token` would also be suitable in my mind.

Reply:
SDK uses the struct `Coin`, which has a `denom` (denomination) and `amount`. I prefer to stick with Coin for consistency